organize/admin.py

```diff
@@ -1,14 +1,155 @@
+        'country',
+        'date',
+        'main_organizer',
+        'status',
+        'comment',
+    )
+    list_filter = ('status',)
+    search_fields = (
+        'city',
+        'country',
+        'main_organizer_first_name',
+        'main_organizer_last_name',
+        'main_organizer_email',
+    )
+    readonly_fields = (
+        'status',
+        'created_at',
+        'status_changed_at',
+        'about_you',
+        'why',
+        'involvement',
+        'experience',
+        'venue',
+        'sponsorship',
+        'coaches'
+    )
     inlines = (InlineCoorganizerAdmin,)
+    suit_form_tabs = (
+        ('general', 'General'),
+        ('application', 'Application'),
+        ('organizers', 'Organizers')
+    )
+    fieldsets = (
+        ('Application info', {
+            'fields': [
+                'status',
+                'created_at',
+                'status_changed_at',
+                'comment'
+            ],
+            'classes': ('suit-tab', 'suit-tab-general',),
+        }),
+        ('Event info', {
+            'fields': [
+                'previous_event',
+                'date',
+                'city',
+                'country',
+                'website_slug',
+            ],
+            'classes': ('suit-tab', 'suit-tab-general',)
+        }),
+        ('Application', {
+            'fields': [
+                'about_you',
+                'why',
+                'involvement',
+                'experience',
+                'venue',
+                'sponsorship',
+                'coaches'
+            ],
+            'classes': ('suit-tab', 'suit-tab-application',)
+        }),
+        ('Main organizer', {
+            'fields': [
+                'main_organizer_email',
+                'main_organizer_first_name',
+                'main_organizer_last_name'
+            ],
+            'classes': ('suit-tab', 'suit-tab-organizers',)
+        })
+    )
+
+    def main_organizer(self, obj):
+        return "{} {} ({})".format(
+            obj.main_organizer_first_name,
+            obj.main_organizer_last_name,
+            obj.main_organizer_email
+        )
+
+    def get_urls(self):
+        urls = super(EventApplicationAdmin, self).get_urls()
+        my_urls = [
+            url(r'(?P<application_id>\d+)/triage/(?P<new_status>[\w\d/]+)/$',
+                self.admin_site.admin_view(self.view_change_application_status),
+                name='organize_eventapplication_change_application_status'),
+        ]
+        return my_urls + urls
+
+    def view_change_application_status(self, request, application_id, new_status):
+        """
+        Custom EventApplication admin view for handling triaging
+        """
+        application = get_object_or_404(EventApplication, id=application_id)
+
+        if new_status in [IN_REVIEW, ON_HOLD]:
+            application.status = new_status
+            application.status_changed_at = timezone.now()
+            application.save()
```

Comment:
Actually, I need to reuse `Event.change_status_to` in this case and add the partial update there.